frontend/src/components/uam/components/group-create/group-create.tsx

```diff
@@ -0,0 +1,86 @@
+import { AppRoute, PaginationDefaultValue } from 'common/enums/enums';
+import { FC, GroupsCreateRequestDto } from 'common/types/types';
+import { Button, Input } from 'components/common/common';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppSelector,
+  useEffect,
+  useNavigate,
+} from 'hooks/hooks';
+import { uamActions } from 'store/actions';
+import { groupCreate } from 'validation-schemas/validation-schemas';
+
+import { DEFAULT_CREATE_GROUP_PAYLOAD } from './common/default-create-group-payload';
+import { GroupCreationFieldsName } from './common/enums/enums';
+import { PermissionsTable } from './components/permissions-table/permissions-table';
+import { UsersTable } from './components/users-table/users-table';
+import styles from './styles.module.scss';
+
+const UAMGroupsCreate: FC = () => {
+  const dispatch = useAppDispatch();
+  const { users, permissions } = useAppSelector((state) => state.uam);
```

Comment:
```suggestion
  const { users, permissions } = useAppSelector((state) => state.groupsCreate);
```
each reducer should be per page